exchanges/gateio/gateio_types.go

```diff
@@ -2689,3 +2689,37 @@ type PriceAndAmount struct {
 	Amount types.Number `json:"amount,omitempty"`
 	Price  types.Number `json:"price,omitempty"`
 }
+
+// BalanceDetails represents a user's balance details
+type BalanceDetails struct {
+	Available      types.Number `json:"available"`
+	Freeze         types.Number `json:"freeze"`
+	Borrowed       types.Number `json:"borrowed"`
+	NegativeLiab   types.Number `json:"negative_liab"`
+	FuturesPosLiab types.Number `json:"futures_pos_liab"`
+	Equity         types.Number `json:"equity"`
+	TotalFreeze    types.Number `json:"total_freeze"`
+	TotalLiab      types.Number `json:"total_liab"`
+	SpotInUse      types.Number `json:"spot_in_use"`
```

Comment:
Missing `funding` and `funding_version`, example json doesn't have it, but you can 🚓